### PR TITLE
Editorial change from 'recorded' flag to 'sampled' flag.

### PR DESCRIPTION
**Pull request by danielkhan**
This addresses #265
The 'recorded' wording when not referring to the flag itself was deliberately left untouched.
Another PR will change the paragraph to be more clear and concise.

### CODE REVIEW

**Review by yurishkuro — APPROVED** [review]

File: spec/20-http_header_format.md [hunk 1]
```diff
@@ -155,21 +155,21 @@ mistake in bit fields is forgetting to mask when interpreting flags.
...
 ```
 
-Current version of specification only supports a single flag called `recorded`.
+Current version of specification only supports a single flag called `sampled`.
 
-#### Recorded Flag (00000001)
+#### Sampled Flag (00000001)
 
-When set, the least significant bit documents that the caller may have recorded
+When set, the least significant bit documents that the caller may have sampled
```
Comment (SergeyKanzhelev) [inline spec/20-http_header_format.md]: one problem with sampled - people not closely familiar with the subject have trouble interpreting "sampled = sampled in" or "sampled = sampled out". Recorded didn't have this problem. Perhaps we can keep the word "recorded" here or specify that sampled means recorded.

File: spec/20-http_header_format.md [hunk 2]
```diff
@@ -186,36 +186,36 @@ specific or application defined.
...
-   be recorded - flag `recorded` should be propagated unchanged. And set to `0`
+   be recorded - flag `sampled` should be propagated unchanged. And set to `0`
    as a default option when trace is initiated by this component. There are two
    additional options:
-    1. Component that makes deferred or delayed recording decision may
-       communicate priority of recording by setting `recorded` flag to `1` for a
-       subset of requests.
-    2. Component may also fall back to probability sampling to set flag
-       `recorded` to `1` for the subset of requests.
+   1. Component that makes deferred or delayed recording decision may
```
Comment (SergeyKanzhelev) [inline spec/20-http_header_format.md]: you broke the sublist. Please make sure to validate your changes using [Editing the spec](https://github.com/w3c/trace-context/blob/master/CONTRIBUTING.md#editing-report) instructions:

image

File: spec/20-http_header_format.md [hunk 3]
```diff
@@ -186,36 +186,36 @@ specific or application defined.
...
 that were marked for recording by caller will also be recorded by SaaS service.
 So caller can troubleshoot the behavior of every recorded request.
 
-Flag `recorded` has no restriction on its mutations except that it can only be
+Flag `sampled` has no restriction on its mutations except that it can only be
 mutated when `parent-id` was updated. See section "Mutating the traceparent
 field". However there are set of suggestions that will increase vendors
 interoperability.
 
 1. If component made definitive recording decision - this decision SHOULD be
```
Comment (SergeyKanzhelev) [inline spec/20-http_header_format.md]: perhaps we can say "sampling decision" than?

Comment (SergeyKanzhelev) [reply spec/20-http_header_format.md]:
```suggestion
1. If component made definitive sampling decision - this decision SHOULD be
```

File: spec/20-http_header_format.md [hunk 4]
```diff
@@ -224,24 +224,24 @@ for future use. Implementations MUST set those to zero.
 
 ### Examples of HTTP headers
 
-*Valid traceparent when caller recorded this request:*
+_Valid traceparent when caller recorded this request:_
```
Comment (SergeyKanzhelev) [inline spec/20-http_header_format.md]: I'm curious why this change?

File: spec/20-http_header_format.md [hunk 5]
```diff
@@ -448,12 +449,12 @@ Here is the list of allowed mutations:
 1. **Update `parent-id`**. The value of property `parent-id` can be set to the
    new value representing the ID of the current operation. This is the most
    typical mutation and should be considered a default.
-2. **Indicate recorded state**. The value of `recorded` flag of `trace-flags`
+2. **Indicate recorded state**. The value of `sampled` flag of `trace-flags`
```
Comment (SergeyKanzhelev) [inline spec/20-http_header_format.md]:
```suggestion
2. **Indicate sampled state**. The value of `sampled` flag of `trace-flags`
```

File: spec/30-processing-model.md [hunk 6]
```diff
@@ -17,40 +17,39 @@ forwards Trace Context headers.
...
-    - The tracer sets the `traceparent` and `tracestate` header for the outgoing
-      request.
+   - The tracing system MUST modify the `traceparent` header.
+     - **Update `parent-id`**. The value of property `parent-id` MUST be set to a
+       value representing the ID of the current operation.
+     - **Update `sampled`**. The value of `sampled` reflects the caller's
+       recording behavior. The value of the `sampled` flag of `trace-flags` MAY
+       be set to `1` if the trace data is likely to be sampled or to `0`
+       otherwise. Setting the flag is no guarantee that the trace will be
+       sampled but increases the likeliness of end-to-end sampled traces.
```
Comment (SergeyKanzhelev) [inline spec/30-processing-model.md]:
```suggestion
       recorded but increases the likeliness of end-to-end recorded traces.
```

It's unclear what end-to-end sampled means. Perhaps recorded is better here

File: spec/20-http_header_format.md [hunk 7]
```diff
@@ -312,9 +312,10 @@ the `OWS` rule from <a data-cite='!RFC7230#section-3.2.3'>RFC7230 section
 
 The `OWS` rule defines an optional whitespace. It is used where zero or more
 whitespace characters might appear. When it is preferred to improve readability
+
```
Comment (SergeyKanzhelev) [inline spec/20-http_header_format.md]: this was not intended to be a list =). It is a dash as a punctuation. Please remove newline.

**Review by SergeyKanzhelev — CHANGES_REQUESTED** [review]
some changes unrelated to PR broke the spec. I think it's some markdown tool you use.

In text I feel there is now a mix or recorded indicating recording and sampled indicating decision. I tried to review where it make sense to keep and where to replace to sampled. I'd advice to re-read again to make sure nothing got left out

File: spec/20-http_header_format.md (hunk 1 above)
Comment (danielkhan) [reply spec/20-http_header_format.md]: Fixed.
